bin/doctest

```diff
@@ -14,8 +14,7 @@ require ('child_process')
   process.execPath,
   [].concat (esmSupported ? ['--experimental-modules'] : [])
     .concat (flags ? args[idx + 1].split (/\s+/) : [])
-    .concat (['--'])
-    .concat ([path.resolve (__dirname, '..', 'lib', 'command')])
+    .concat (['--', path.resolve (__dirname, '..', 'lib', 'command')])
```

Comment:
What do you think? https://github.com/davidchambers/doctest/pull/113/commits/fa5df0e0c174c613432546b875800f6a1bc24e77

Reply:
:heart: